server/SnowplowMicroServer.js

```diff
@@ -0,0 +1,64 @@
+const axios = require("axios");
+const path = require("path");
+const { spawn } = require("child_process");
+
+class SnowplowMicroServer {
+    constructor(appPath) {
+        this.appPath = appPath;
+    }
+
+    start() {
+        const appPath = this.appPath.replace("app.asar", "");
+        const jarPath = path.join(appPath, "jars", "snowplow-micro-1.3.4.jar");
+        const microConfPath = path.join(appPath, "snowplow_micro_config", "micro.conf");
+        const igluPath = path.join(appPath, "snowplow_micro_config", "iglu.json");
+        const command = spawn("java", [
+            "-jar",
+            jarPath,
+            "--collector-config",
+            microConfPath,
+            "--iglu",
+            igluPath,
+        ]);
+
+        // Figure out on which port Snowplow Micro is running
+        command.stderr.on("data", (data) => {
+            const match = data.toString()?.match(/.*:(\d*).?/i);
+            if (match && match[1]) {
+                console.log(`Snowplow Micro is running on port: ${match[1]}`);
+                this.port = match[1] ?? this.port;
```

Comment:
Should we update `esversion` [here](https://github.com/PicnicSupermarket/nepsnowplow/blob/9c4d5d543da6d2a4aa4a2cb184de8bef9178173e/.jshintrc#L20) to esversion 11 to avoid warnings related to `??`

![Screenshot 2022-12-20 at 10 20 56](https://user-images.githubusercontent.com/14923445/208630582-0683e194-2c1e-4990-9315-18ce9d14b323.png)